test/msw-api/res/network-error.mocks.ts

```diff
@@ -0,0 +1,10 @@
+import { setupWorker, rest } from 'msw'
+
+const worker = setupWorker(
+  rest.get('/user', (_, res) => {
+    return res.networkError()
+  }),
+)
+
+// @ts-ignore
+window.__MSW_START__ = worker.start
```

Comment:
You don't have to delegate the start like this (I believe this is done only for specific tests). Please, call `worker.start()` directly.

Reply:
I changed this in the latest commit